### PR TITLE
Feature/ListRoom

### PR DESCRIPTION
**Pull request by Hazel1603**
Implement ListRoom Command.

### CODE REVIEW

**Review by iamgenechua — COMMENTED** [review]
looks good. Will be good to merge once checkstyle is cleaned up

**Review by shermz-lim — APPROVED** [review]
lgtm but have to fix merge conflict first before merging

**Pull request merged**
